File: C#/0009. Palindrome Number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if(x<0){
            return false;
        }
        int target=x;
        int rep=0;
        while(x>0){
            int tmp=x%10;
            rep=rep*10+tmp;
            x=x/10;
        }
        return rep==target;
    }
}

File: 20200215Python-China/0009. Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        if x==0:
            return True
        if x%10==0:
            return False
        y=0
        tmp=x
        while tmp>0:
            y=y*10+tmp%10
            tmp=(tmp-tmp%10)//10
        return x==y
